NEWS.md

```diff
@@ -1,5 +1,7 @@
 # chevron 0.2.5.9005
 
+* Assert `summaryvars` class in `dmt01`.
```

Comment:
This looks like a "summary class" similarly to "character class" which I don't think is what you wanted :)

```suggestion
* Added assertion on class of `summaryvars` argument of `dmt01()`.
```